packages/image_picker/ios/Classes/FLTImagePickerMetaDataUtil.m

```diff
@@ -63,14 +63,16 @@ + (NSData *)updateMetaData:(NSDictionary *)metaData toImage:(NSData *)imageData
 + (NSData *)convertImage:(UIImage *)image
                usingType:(FLTImagePickerMIMEType)type
                  quality:(nullable NSNumber *)quality {
-  if (quality && type != FLTImagePickerMIMETypeJPEG) {
-    @throw [NSException
-        exceptionWithName:@"flutter_image_picker_convert_image_exception"
-                   reason:[NSString stringWithFormat:@"quality is not available for type %@",
-                                                     [FLTImagePickerMetaDataUtil
-                                                         imageTypeSuffixFromType:type]]
-                 userInfo:nil];
-  }
+
+  // Commenting as of now as JPEG conversion is done by default
```

Comment:
We should still keep this check. However, we don't want to crash it anymore. Since the compressing on PNG with quality is not supported. We want to let the plugin users aware of it. 
Something like 
```objc
if (quality && type != FLTImagePickerMIMETypeJPEG) {
   NSLog(@"image_picker: compressing is not supported for type %@. Returning the image with original quality");
}

```

This is mainly to handle when plugin users choose to reduce quality but the user picked a PNG image.

Reply:
done